doc/contributing/writing-tests.md

```diff
@@ -139,6 +139,28 @@ request. Interesting things to notice:
   exit gracefully. Remember that for a test to succeed, it must exit with a
   status code of 0.
 
+## Running tests
+
+All tests can be executed by running the `jstest` target:
+
+```bash
+make jstest
+```
+
+A filter can be applied to only run specific tests:
+
+```bash
+make jstest test/parallel/test-debugger-pid.js test/sequential/test-debugger-pid.js test/sequential/test-debugger-launch.mjs
+```
```

Comment:
I don't think that's true, it runs the full test suite for me.

````suggestion
A filter can be applied to only run specific tests:

```bash
./tools/test.py test/parallel/test-debugger-pid.js test/sequential/test-debugger-pid
```
````